shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,75 @@
+
+But if nobody opened up their inbound connection through their firewall, then all these dcrd nodes would be reaching out to the IP addresses in their peers list and asking "hey, can I talk to you? Do you have new blocks?" And the firewall would just block all the traffic and no peers would be able to talk to each other.
+
+The point is that somebody has to open up their inbound connection to allow their node to be publicly accessible. The more people that do this will increase the security and decentralisation of the network. 
+
+## Other nodes in the network
+Decred has a multitude of services that run dcrd nodes in one way or another. Some of these options also include incentives for participation, which in turn encourages users to run a top tier fully validating node. What options do we have?
+ 
+**Decrediton** can be run either as a fully validating node or in SPV mode. For security reasons, wallets will never have an open inbound connection. Wallets running in fully validating mode allow for maximum personal privacy and security. On the other hand, SPV mode is great for mobile wallets or when a device has limited resources. 
+
+**Lightning Network Channels** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. LN operators are also incentivised with the fees from conducting transactions over the Lightning Network. For the best performance, these nodes should run constantly 24/7/365.
+
+**Voting Service Providers (VSP)** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. VSP operators are also incentivised with the fees for conducting ticket transactions. For the best performance, these nodes should run constantly 24/7/365 and have +4 instances to increase reliability.
+
+**DCRDEX Operators** need access to trusted full nodes for each of the assets supported. While operation via a surrogate blockchain data service such as a block explorer is potentially feasible, it would entail significant security risks. 
+
+?? What other services could I add, and how do they interact with nodes ??
+
+## Conclusion
+Running a fully validating node is the best way of understanding how the network is performing. When you participate in the decentralisation of the network, you understand what it takes to run it and the levels to which you are contributing. 
+
+It occurs to me, from my research here, that decentralisation is a collective endeavour, and making the system and your investment more secure means your participation is necessary. In centralised products, you can rely on someone else to run the system. In fully decentralised products, you are the system and the responsibility sits squarely at your feet.
```

Comment:
more sp: `centralized` and `decentralization`

Reply:
Haha, perhaps! Not a big deal if y'all spell it differently in the UK!

Reply:
There's also `initialises` above so I assume it is UK, so `centralised` and `decentralisation`. Any flavor is fine IMO (and it is somewhat fun to have different styles around), but at least keep it internally consistent - full post US or full post UK but not a mix.

Reply:
I agree. Keep the UK spellings! It is fun to have the different styles.